quickwit/quickwit-search/src/leaf.rs

```diff
@@ -345,8 +345,8 @@ async fn leaf_search_single_split(
         quickwit_collector.requires_scoring(),
     )
     .await?;
+    let span = info_span!( "tantivy_search", split_id = %split.split_id);
```

Comment:
We lose the link to the parent span if the span is in placed in `run_cpu_intensive`.